velox/functions/prestosql/tests/ArithmeticTest.cpp

```diff
@@ -603,6 +603,12 @@ TEST_F(ArithmeticTest, toBase) {
   EXPECT_EQ("20a2", to_base(3578, 12));
   EXPECT_EQ("-20a2", to_base(-3578, 12));
 
+  EXPECT_EQ(
```

Comment:
Interesting find: I am not able to replicate this in presto, looks like presto does not accept int64:min() as a valid bigint:
Tried these:
`SELECT TO_BASE(-9223372036854775808, 5);`
`SELECT TO_BASE(cast(-9223372036854775808 as bigint), 5);`

Reply:
you can do this in presto 
select to_base(cast ('-9223372036854775807' as bigint), 5)

Reply:
select to_base(cast ('-9223372036854775808' as bigint), 5)**